packages/nl-design-system-unstable/__docs__/form-patterns.md

```diff
@@ -1,139 +1,270 @@
+### Meerstappen formulieren
 
-If the form is long or contains a lot of information and becomes difficult for users to process. Consider breaking it down into multiple steps or pages: ‘[chunking](https://www.nngroup.com/articles/chunking/)’.
+Als het om een beperkt aantal onderwerpen gaat die onderverdeeld kunnen worden in een klein aantal secties dan kunnen deze op een enkele pagina getoond worden.
 
-## Text labels
+Als het formulier lang is of veel informatie bevat en hierdoor lastig te interpreteren is voor bezoekers, overweeg dan deze op te splitsen in meerdere stappen en/of pagina's. Zorg er echter voor dat deze stappen in betekenisvolle groepen zijn verdeeld.
 
-A text label allows the user to scan a form clearly and comfortably. All form elements must be accompanied by a textual label; a checkbox, select menu, radio button, textarea, etc.
+Als het lastig blijkt het formulier te interpreteren als niet alle vragen op een enkel scherm voorhanden zijn, splits deze dan niet op in meerdere stappen.
 
-A proper label is placed outside of the input field, not within it. This is naturally the case with most form elements; a checkbox, for example, has no room to internally accommodate a text label. For buttons, this label is the button text itself; a button performs an action and is not an element where a user can enter data.
+#### Navigatie en oriëntatie
 
-Several studies have shown that the best position for a text input label is directly top left of an input field (for LTR languages). In this way, the label always has a fixed position and can be displayed in a readable text size. Because of this fixed position, a form can be easily scanned and a user can immediately distinguish between a field that has been filled in and a field that still needs to be filled in.
+Wanneer een formulier uit meerdere stappen en/of pagina’s bestaat dan dient deze voorzien te zijn van navigatie. Deze informeert over het totale aantal stappen, de huidige positie binnen deze stappen, en biedt een manier om hiertussen te navigeren. Bij het gebruik van meerdere pagina’s dient de ingevoerde informatie tussen pagina’s opgeslagen te worden en is deze mogelijk te wijzigen door de bezoeker.
 
-### Interaction target area
+### Conditionele velden
 
-An additional benefit of placing a label outside an input field is that you make the click and tap area larger making selecting an input field easier for both mouse and touchscreen users. A properly implemented label has an association with the form element; web browsers allow users to set focus to an input element by clicking on the text label.
+Conditionele velden of ‘selectie-afhankelijke vragen’ zijn formulierelementen die getoond worden als reactie op een eerdere keuze of invoer.
```

Comment:
Ik denk dat we hier aan toe willen voegen dat deze conditionele velden altijd **na** het element moeten komen waar aan de conditie wordt voldaan.

Reply:
Ja, goed punt, ik had zoiets ook al opgeschreven n.a.v. de laatste formulierensessie. Neem ik mee in de documentatie!